packages/core/src/electron-main/electron-main-application.ts

```diff
@@ -54,19 +54,8 @@ export interface ElectronMainCommandOptions {
      */
     readonly file?: string;
 
-}
-
-/**
- * Fields related to a launch event.
- *
- * This kind of event is triggered in two different contexts:
- *  1. The app is launched for the first time, `secondInstance` is false.
- *  2. The app is already running but user relaunches it, `secondInstance` is true.
- */
-export interface ElectronMainExecutionParams {
-    readonly secondInstance: boolean;
-    readonly argv: string[];
     readonly cwd: string;
+    readonly secondInstance: boolean;
```

Comment:
I think we should transfer the type doc comments for `secondInstance`, as it may not be obvious what that means.